test/devnet-create-block.test.ts

```diff
@@ -10,11 +10,13 @@ describe("Devnet create block", function () {
         const latestBlock = await starknet.getBlock();
 
         const emptyBlock1 = await starknet.devnet.createBlock();
-        expect(emptyBlock1.block_number).to.deep.equal(latestBlock.block_number + 1);
-        expect(emptyBlock1.transactions).to.be.empty;
+        expect(Number(emptyBlock1.block_hash)).to.deep.equal(Number(latestBlock.block_hash) + 1);
```

Comment:
Why replace block_number with block_hash? And why does this block hash equation hold?

We might be better off with requesting block creation, then getting the block and just asserting that it's empty - i.e. not holding any txs). I.e. what we already have, but remove the hash equality assertion.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/104/commits/8d02a690f9f40afe9033bdbf699e82de2789c5a9